package.json

```diff
@@ -48,7 +49,7 @@
 		"@types/jsdom": "^20.0.0",
 		"@types/mocha": "^9.1.1",
 		"@types/node": "^20.2.5",
-		"@types/prettier": "^2.7.0",
+		"@types/prettier": "^2.7.3",
```

Comment:
Shouldn't we use the same version as Prettier, at least v3.x.y? Or is it not necessary in this case?

Reply:
Good catch - I don't think it's necessary anymore since I switched over to running prettier sync.  It looks like `@prettier/sync` ships with its own type defintion.